src/components/picker/index.tsx

```diff
@@ -222,15 +271,18 @@ const Picker = (props: PropsWithChildren<PickerProps> & ForwardRefInjectedProps
             ref={pickerRef}
             // {...textInputProps}
             {...others}
+            {...propsByFieldType}
             testID={`${testID}.input`}
-            containerStyle={[paddings, margins, positionStyle, containerStyle]}
+            containerStyle={[paddings, margins, positionStyle, containerStyle, propsByFieldType?.containerStyle]}
             {...accessibilityInfo}
             importantForAccessibility={'no-hide-descendants'}
             value={label}
             selection={Constants.isAndroid ? {start: 0} : undefined}
             /* Note: Disable TextField expandable feature */
             topBarProps={undefined}
-          />
+          >
+            {renderPickerInnerInput()}
```

Comment:
It feels a little bit an overkill to render the `filter` and `settings` types under a `TextField`, WDYT?